### ISSUE TITLE
About Model

### ISSUE THREAD

**Issue by tikboaHIT** [navervision/lincir#8]
Nice work! Do you have a plan for when you will release the model?

**Comment by geonm** [comment]
Certainly. We haven't released the models used for writing our paper, but we have made a model for demo available.

You can find what you need here:
[LinCIR Demo](https://github.com/navervision/lincir/blob/master/demo.py#L69-L74).

**Comment by tikboaHIT** [comment]
Great. Can this demo model achieve the accuracy of the paper?

**Comment by geonm** [comment]
Sure.

Here are the results

### FashionIQ

model name | Shirt R@10 | Shirt R@50 | Dress R@10 | Dress R@50 | Toptee R@10 | Toptee R@50 | Avg R@10 | Avg R@50
-- | -- | -- | -- | -- | -- | -- | -- | --
Ours_paper | 29.10 | 46.81 | 20.92 | 42.44 | 28.81 | 50.18 | 26.28 | 46.49
Ours_for_demo | 28.66 | 47.11 | 20.67 | 42.34 | 28.96 | 49.31 | 26.10 | 46.25

### CIRR

model name | Full R@1 | Full R@5 | Full R@10 | Subset R@1 | Subset R@2 | Subset R@3
-- | -- | -- | -- | -- | -- | --
Ours_paper | 25.04 | 53.25 | 66.68 | 57.11 | 77.37 | 88.89
Ours_for_demo | 25.398 | 53.301 | 67.036 | 56.241 | 76.94 | 88.578

### CIRCO

model name | mAP@5 | mAP@10 | mAP@25 | mAP@50
-- | -- | -- | -- | --
Ours_paper | 12.59 | 13.58 | 15.00 | 15.85
Ours_for_demo | 12.75 | 13.63 | 14.99 | 15.90

**Comment by sungonce** [comment]
Hi @geonm!
Thanks for sharing your research. It's really interesting. 
I tried reproducing your results using the training script in the readme file, but my results were slightly lower than the demo weight you provided. Could you share the exact script that you used to train the demo weight?

**Comment by geonm** [comment]
Thank you for your interest!

How about it?

Please change the following:
```
--validation_steps 1000 \
--checkpointing_steps 1000 \
```
to
```
--validation_steps 500 \
--checkpointing_steps 500 \
```
It would validate the Phi in a shorter period and you may find slightly improved Phi model.

Please note that even you make the validation steps shorter and get the improved model, it is possible to show "slightly" lower performance in terms of other benchmarks.

We believe that all the results you've got should be almost consistent despite differences between models you've trained.

**Comment by sungonce** [comment]
Thanks for your quick response.
I'll try it again with your suggestions.

I have trained your proposed model multiple times and find it to be very robust and consistent.

Thanks again for sharing interesting research!
